actionmailer/test/fixtures/another.path/base_mailer/welcome

```diff
@@ -0,0 +1 @@
+Welcome from another path
```

Comment:
If add a new line, tests fail.
Do you think it is better to add a new line and change a test case?

```diff
diff --git a/actionmailer/test/fixtures/another.path/base_mailer/welcome b/actionmailer/test/fixtures/another.path/base_mailer/welcome
index ef45129..95079a9 100644
--- a/actionmailer/test/fixtures/another.path/base_mailer/welcome
+++ b/actionmailer/test/fixtures/another.path/base_mailer/welcome
@@ -1 +1 @@
-Welcome from another path
\ No newline at end of file
+Welcome from another path
```

```shell
  1) Failure:
MailerCustomFixturePathTest#test_custom_fixture_path [rails/actionmailer/test/test_case_test.rb:43]:
--- expected
+++ actual
@@ -1 +1,2 @@
-["Welcome from another path"]
+["Welcome from another path
+"]
```

To pass tests

```diff
--- a/actionmailer/test/test_case_test.rb
+++ b/actionmailer/test/test_case_test.rb
@@ -40,6 +40,6 @@ class MailerCustomFixturePathTest < ActionMailer::TestCase
   self.fixture_path = FIXTURE_LOAD_PATH + "/another.path"

   def test_custom_fixture_path
-    assert_equal ["Welcome from another path"], read_fixture('welcome')
+    assert_equal ["Welcome from another path\n"], read_fixture('welcome')
   end
 end
```